packages/design-system/src/components/N8nActionBox/ActionBox.vue

```diff
@@ -15,13 +15,19 @@
 				</slot>
 			</N8nText>
 		</div>
-		<N8nButton
-			v-if="buttonText"
-			:label="buttonText"
-			:type="buttonType"
-			size="large"
-			@click="$emit('click:button', $event)"
-		/>
+		<N8nTooltip :disabled="!buttonDisabled">
+			<template #content>
+				<slot name="disabledButtonTooltip"></slot>
```

Comment:
I think this slot should be usable for non-disabled buttons as well. Can you name it `buttonTooltip` instead?